base/methodshow.jl

```diff
@@ -131,9 +131,13 @@ const methodloc_callback = Ref{Union{Function, Nothing}}(nothing)
 function fixup_stdlib_path(path::String)
     # The file defining Base.Sys gets included after this file is included so make sure
     # this function is valid even in this intermediary state
-    if isdefined(@__MODULE__, :Sys) && Sys.BUILD_STDLIB_PATH != Sys.STDLIB::String
-        # BUILD_STDLIB_PATH gets defined in sysinfo.jl
-        path = replace(path, normpath(Sys.BUILD_STDLIB_PATH) => normpath(Sys.STDLIB::String))
+    if isdefined(@__MODULE__, :Sys)
+        BUILD_STDLIB_PATH = Sys.BUILD_STDLIB_PATH::String
+        STDLIB = Sys.STDLIB::String
```

Comment:
```suggestion
        BUILD_STDLIB_PATH = Sys.BUILD_STDLIB_PATH
        STDLIB = Sys.STDLIB
```

Reply:
I think we want to keep these annotations since his method can be compiled before `sysinfo.jl` is loaded and `Sys` is defined and so these global references can be `Any` at that moment AFAIU.